### DISCUSSION TITLE
UI Changes and ../scripts dir

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/thorium#7] [Ideas]
@pierro78 @midzer I'm thinking about changing the WebUI tabstrip to use red buttons. There was a bug in the windows clang compiler that caused this a lil while back, but I actually liked the look. So I'm thinking about intentionally adding it. What do ya'll think.

Also, the scripts for building thorium yourself are currently in the root dir of the repo, do you think I should move them to a scripts dir? There's only three of them (trunk.sh for syncing the source and downloading the PGO profile, setup.sh for patching/copying the thorium sources over the chromium tree, and clean.sh for cleaning your build dir of artifacts if one wants to use it without the .deb)

**Comment by badbucket1301** [comment]
Hi. I witnessed that bug in one of [Robert](https://github.com/RobRich999/Chromium_Clang)'s builds and found it ugly as sin. Maybe a custom accent color or different red tone might be worth a try but the way the bug "implemented" that "feature" was just very hard on my eyes.

And I think it is a good idea to create a new sub dir for those scripts after having a glance at the current structure.
(Take this with a grain of salt as I am no dev or github guru by any stretch of the imagination)

**Comment by Alex313031** [comment]
@einsnulleins I found changing the tabstrip colour too much work, and thought it would kind of be "forcing" something on users since they wouldn't be able to change it. So I'm not going forward with that. And about the scripts dir, I think I'm actually going to leave them as is, because the scripts would have to be re written since they all assume they're is in the root of the thorium repo and that the chromium repo is side by side with the thorium repo in $HOME. And the readme would have to be re written in the building section. Doing all that just for the "aesthetics" of having them in a scripts dir doesnt make sense to me. If this project had a whole bunch of scripts, or any of the scripts did things on subcomponents of the project rather than the project as a whole for building it, then it might make more sense but.

**Comment by Alex313031** [comment]
@einsnulleins @midzer @gz83 Marking as completed, as the infra subdir and win_scripts dir serves these purposes, and I have updated icon colours in Thorium, but decided not to go with an overall "red" theme as I thought that was too much as per above.

**Comment by gz83** [comment]
It seems this discussion is outdated.
